refarch-integrations/refarch-s3-integration/pom.xml

```diff
@@ -20,6 +20,8 @@
         <module>refarch-s3-integration-core</module>
         <module>refarch-s3-integration-starter</module>
         <module>refarch-s3-integration-client-core</module>
+        <!-- Java -->
```

Comment:
Do we need those comments if it only includes 1 module? I think we should remove those. ;)

Reply:
Done